packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -3754,6 +3785,10 @@ export interface NotificationStackStrings {
     stopScreenShareGeneric?: NotificationStrings;
     stopVideoGeneric?: NotificationStrings;
     teamsMeetingCallNetworkQualityLow?: NotificationStrings;
+    // (undocumented)
```

Comment:
won't comment on each one, but we aim to have no `// (undocumented)` public functions